src/components/DuosFooter.js

```diff
@@ -1,32 +1,23 @@
-import { Component } from 'react';
 import { a, div, footer, img, li, ul } from 'react-hyperscript-helpers';
 import { Storage } from '../libs/storage';
 
+const DuosFooter = (props) => {
+  const isAdmin = props.isLogged ? Storage.getCurrentUser().isAdmin : false;
```

Comment:
You could clean this up a bit. For example, `Storage.getCurrentUser()` returns null if it isn't set. So, instead of relying on props, you could just call that and check if null, then look for the `isAdmin` property. I like lodash/functional lodash for this kind of thing.
* Lodash equivalent: https://lodash.com/docs/#isEmpty
* https://github.com/lodash/lodash/wiki/FP-Guide